aws/rust-runtime/aws-config/src/sts/assume_role.rs

```diff
@@ -154,24 +188,24 @@ impl AssumeRoleProviderBuilder {
             .set_role_arn(Some(self.role_arn))
             .set_external_id(self.external_id)
             .set_role_session_name(Some(session_name))
+            .set_duration_seconds(self.session_length.map(|dur| dur.as_secs() as i32))
```

Comment:
it'd be kinda neat if we had customizations that provided `into` impls for these "duration-like" fields. I think we'd have to do it manually though